setup.py

```diff
@@ -313,14 +316,22 @@ def _get_webhost_version() -> str:
         return latest[0]['name'].replace('v', '')
 
     @staticmethod
-    def _download_webhost_zip(version: str) -> str:
+    def _download_webhost_zip(version: str, branch: str) -> str:
         # Return the path of the downloaded host
         temporary_file = tempfile.NamedTemporaryFile()
-        zip_url = (
-            'https://github.com/Azure/azure-functions-host/archive/'
-            f'v{version}.zip'
-        )
-        print(f'Downloading Functions Host ({version}) from {zip_url}')
+
+        if branch is not None:
+            zip_url = (
+                'https://github.com/Azure/azure-functions-host/archive/'
```

Comment:
Minor thing: Can you Constantify the first part of the URL?

Reply:
+1